src/components/dialogs/create-study-dialog.js

```diff
@@ -212,11 +212,11 @@ export const CreateStudyDialog = ({ open, onClose, providedCase }) => {
         useImportExportParams(formatWithParameters);
 
     const [
-        selectedFile,
+        providedCaseFile,
```

Comment:
Why renaming ?

Reply:
It wasn't clear between provided, selected, etc... and wasn't easy to know what exactly are these variables.
The file is not "selected" but "provided" that's why I renamed it.